Jenkinsfile

```diff
@@ -42,12 +42,13 @@ def buildDetails = { vars ->
   """.stripIndent()
 }
 
-def notify = { message, color='good' ->
+def notify = { ->
   if (env.BRANCH_NAME == devBranch ||
       env.BRANCH_NAME == stagingBranch ||
       env.BRANCH_NAME == prodBranch) {
+    message = "vets-website ${env.BRANCH_NAME} branch CI failed. |${env.RUN_DISPLAY_URL}".stripMargin()
```

Comment:
neato, did not know about RUN_DISPLAY_URL

Reply:
Yeah it links the Blue Ocean UI which I think is nicer for the vets-website build. Turns out the Jenkins docs around it are still not great.